.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: cmpgeos
       run: make cmpgeos
     - name: Convert coverage to lcov
-      uses: jandelgado/gcov2lcov-action@v1.0.2
+      uses: jandelgado/gcov2lcov-action@v1.0.9
```

Comment:
Needed to get the latest gcov2lcov since the older one isn't aware of the retract directive.